packages/gtk3.rb

```diff
@@ -41,34 +30,35 @@ class Gtk3 < Package
   depends_on 'graphite'
```

Comment:
`depends_on 'llvm' => :build` is also a core package.

Reply:
Removed